### PR TITLE
Service Bus extension API updates

### PR DESCRIPTION
**Pull request by JoshLove-msft**
Updates based on API review comments - https://apiview.dev/Assemblies/Review/88f79b19a3894a748e3f4f367ea363ab

### CODE REVIEW

**Review by pakrym — APPROVED** [review]

File: sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +1,11 @@
...
 
 ### Breaking Changes
-
-### Key Bugs Fixed
-
-### Fixed
-
+- Renamed `ServiceBusEntityType` property to `EntityType`.
+- Renamed `messageActions` and `sessionActions` parameters to `actions` in `MessageProcessor` and `SessionMessageProcessor`.
+- Renamed `MaxBatchSize` to `MaxMessageBatchSize` in `ServiceBusOptions`.
```
Comment (JoshLove-msft) [inline sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/CHANGELOG.md:8]: @pakrym, @jsquire do we want to rename the equivalent in EH to `MaxEventBatchSize`?

**Review by jsquire — APPROVED** [review]

File: sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/CHANGELOG.md (hunk 1 above)
Comment (jsquire) [reply sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/CHANGELOG.md:8]: I wasn't in the board reviews, so please take my thoughts with a grain of salt... but I think that makes sense.

Comment (JoshLove-msft) [reply sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/CHANGELOG.md:8]: This was just based on a comment in the API review from @KrzysztofCwalina

Comment (pakrym) [reply sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/CHANGELOG.md:8]: I think it makes sense.

**Pull request merged**